app/routes/search_label.js

```diff
@@ -2,6 +2,14 @@ import AuthorizeRoute from './authorize';
 
 export default AuthorizeRoute.extend({
 
+  afterModel(model, transition){
+    var pkgs = model.get("packages");
+    if(pkgs && pkgs.length > 0 && (pkgs.get("firstObject.hasAllPackagesDesignated") || pkgs.get("firstObject.hasAllPackagesDispatched"))){
+      transition.abort();
+      this.get('messageBox').alert(this.get("i18n").t('designated_dispatched_error'));
```

Comment:
We are staying on the same page for this case i.e offers/11/review_item/21/accept page after we press okay on this pop up.